docs/tutorial_email_generator.md

```diff
@@ -0,0 +1,225 @@
+
+But my absolute favorite part was the food. I ate so much ramen, it was delicious!
+
+Catch up soon,
+
+[Your Name]
+```
+```text
+Subject: Running late for the 10am meeting!
+
+Hi Kenny,
+
+I just wanted to drop you a quick email to let you know that I will be about 10 minutes late for the 10am meeting this morning. I messed up a train transfer and it set me back a bit. I apologize for any inconvenience caused.
+
+See you shortly.
+
+Best,
+[Your Name]
+```
+
+Based on these couple examples, it seems like the app is doing a pretty good job. However, it's not always perfect - in the example below, the app "hallucinates" a bunch of fake qualifications that we did not specify!
+
+```python
+subject = "Application for the software engineer role"
+recipient_name = "Google"
+content_description = """
+- I want to apply for the role
+- I have many years of experience in AI, making me a strong candidate
+- Ready to start ASAP
+"""
+email = generate_email(subject, recipient_name, content_description)
+print(email)
+```
+```text
+Subject: Application for the Software Engineer Role
+
+Dear Google Hiring Team,
+
+I hope this email finds you well. I am writing to express my interest in applying for the Software Engineer role at Google.
+
+With several years of experience specializing in AI, I believe I am a strong candidate for this position. Throughout my career, I have developed a deep understanding and proficiency in various AI technologies and their application in real-world scenarios. My expertise spans across machine learning, natural language processing, and computer vision, among other domains. I am well-versed in programming languages such as Python, Java, and C++, and have a solid foundation in data structures and algorithms.
+
+Having closely followed Google's groundbreaking advancements in AI and software engineering, I am particularly enthused about the opportunity to contribute to such innovative projects. I am confident that my experience, skills, and enthusiasm would be valuable assets to the team.
+
+Additionally, I am ready to start as soon as possible and eager to hit the ground running. I am highly adaptable, possess excellent problem-solving abilities, and thrive in collaborative environments. Moreover, I am a quick learner, constantly seeking ways to stay updated with the latest trends and technologies in the field of AI.
+
+Thank you for considering my application. I look forward to the possibility of joining Google and contributing to its mission of pushing the boundaries of technology.
+
+Sincerely,
+[Your Name]
+```
+
+## Evaluating with LangCheck (No References)
+
+At this point in our development process of the email generator, we would start iterating on the prompt template to try and improve the outputs. This is where LangCheck can come in and help - rather than having to manually prompt the app and check the outputs each time, we can automatically evaluate the outputs with the various LangCheck metrics!
+
```

Comment:
Edited to be more concise

```suggestion
At this point in our development process, we would start iterating on the prompt template to try and improve the outputs. This is where LangCheck can come in and help - rather than having to manually prompt the app and check the outputs each time, we can automatically evaluate the outputs with the various LangCheck metrics!
```